SCRUM-2: install mongoose and uninstall prisma

Jomalagon opened the pull request:
@axelzo 

I have installed Mongoose and removed Prisma from the project. Everything looks good.

Jomalagon committed:
SCRUM-2: install Mongoose

axelzo reviewed (changes requested):
Nice work!
Two observations:
1- Follow the common commit pattern, try not using Title Case or special characters in the commit, please update the commit  message for : `SCRUM-2: install mongoose and uninstall prisma`.
For acomplish this task use `git commit --amend` and use the vim editor to modify the message.
2- It is not deleted `prisma/client` package in the `package.json`
 ```
 "dependencies": {
    "@prisma/client": "^6.18.0",
```
and also it i not updated the `postinstall` script.

```
  "scripts": {
    "postinstall": "prisma generate",

```
thank you!

Jomalagon commented:
@axelzo 
I changed the name of the commit and noticed that the prisma was not uninstalled. I will add the change in the next commit.

Jomalagon committed:
migrate database to mongodb

Jomalagon commented:
I have completed the database migration, configured the database connection, created the Mongoose models, refactored the controllers and endpoints, and removed references to Prisma.
